updating the cookie policy (#3396)
DFE-Digital/get-into-teaching-app: app/views/pages/cookies.md

---
title: Cookies Policy
---

Last updated: 12 July 2023

> This Cookies Policy explains what cookies are and how we use them. You should read this policy so you can understand what type of cookies we use, or the information we collect using cookies and how that information is used.

Reviewer: ```suggestion
This Cookies Policy explains what Cookies are and how we use them. You should read this policy so you can understand what type of Cookies we use, or the information we collect using Cookies and how that information is used.
```

> Cookies do not typically contain any information that personally identifies a user, but personal information that we store about you may be linked to the information stored in and obtained from cookies. For further information on how we use, store and keep your personal data secure, see our [Privacy Policy](/privacy-policy).

Reviewer: ```suggestion
Cookies do not typically contain any information that personally identifies a user, but personal information that we store about you may be linked to the information stored in and obtained from Cookies. For further information on how we use, store and keep your personal data secure, see our [Privacy Policy](/privacy-policy).
```

> We want to make our services easy, useful and reliable. When services are delivered on the internet, this sometimes involves placing small amounts of information on your computer, tablet, or mobile phone – whichever device you use to access our site. These include small files known as cookies.

Reviewer: ```suggestion
We want to make our services easy, useful and reliable. When services are delivered on the internet, this sometimes involves placing small amounts of information on your computer, tablet, or mobile phone – whichever device you use to access our site. These include small files known as Cookies.
```

> We do not store sensitive personal information, such as mailing addresses, account passwords, etc. in the cookies we use.

Reviewer: ```suggestion
We do not store sensitive personal information, such as mailing addresses, account passwords, etc. in the Cookies we use.
```

## Interpretation and definitions

### Interpretation

The following definitions shall have the same meaning regardless of whether they appear in singular or in plural.

### Definitions

For the purposes of this Cookies Policy:

> - 'Department for Education' (referred to as either "DfE", " we ", "us" or "our" in this Cookies Policy) refers to Department for Education

Reviewer: ```suggestion
- 'Department for Education' (referred to as either "DfE", " we", "us" or "our" in this Cookies Policy) refers to Department for Education
```

> - 'cookies' means small files that are placed on your computer, mobile device or any other device by a website, containing details of your browsing history on that website among its many uses

Reviewer: ```suggestion
- 'Cookies' means small files that are placed on your computer, mobile device or any other device by a website, containing details of your browsing history on that website among its many uses
```

> - 'website' refers to Get Into Teaching, accessible from https://getintoteaching.education.gov.uk/

Reviewer: ```suggestion
- 'Website' refers to Get Into Teaching, accessible from https://getintoteaching.education.gov.uk/
```

> - 'you' means the individual accessing or using the website, or a company, or any legal entity on behalf of which such individual is accessing or using the website, as applicable

Reviewer: ```suggestion
- 'You' means the individual accessing or using the Website, or a company, or any legal entity on behalf of which such individual is accessing or using the Website, as applicable
```

> ## The use of the Cookies

Reviewer: ```suggestion
## The use of the cookies
```

> ## The use of the cookies

Reviewer: ```suggestion
## The use of the Cookies
```

> The information below shows the cookies that are set by [education.gov.uk](https://www.gov.uk/government/organisations/department-for-education) and the third-party services we use. You may refuse the use of cookies by selecting the appropriate settings on your browser. However, please note that if you do this you may not be able to use the full functionality of this website.

Reviewer: ```suggestion
The information below shows the Cookies that are set by [education.gov.uk](https://www.gov.uk/government/organisations/department-for-education) and the third-party services we use. You may refuse the use of Cookies by selecting the appropriate settings on your browser. However, please note that if you do this you may not be able to use the full functionality of this Website.
```

> We are committed to following the relevant privacy and electronic communication regulations. At the footer of this page, you can change your preferences at any time for marketing, tracking or performance cookies.

Reviewer: ```suggestion
We are committed to following the relevant privacy and electronic communication regulations. At the footer of this page, You can change your preferences at any time for marketing, tracking or performance Cookies.
```

## Types of cookies we use

> Cookies can be "persistent" or "session" cookies. Persistent cookies remain on your personal computer or mobile device when you go offline, while session cookies are deleted as soon as you close your web browser.

Reviewer: ```suggestion
Cookies can be "Persistent" or "Session" Cookies. Persistent cookies remain on your personal computer or mobile device when you go offline, while Session Cookies are deleted as soon as you close your web browser.
```

### Essential cookies

**Type:** Session cookies

**Administered by:** Us

> **Purpose:** These cookies are essential to provide you with services available through the website and to enable you to use some of its features. They help to authenticate users and prevent fraudulent use of user accounts. Without these cookies, the services that you have asked for cannot be provided, and we only use these cookies to provide you with those services.

Reviewer: ```suggestion
**Purpose:** These Cookies are essential to provide you with services available through the Website and to enable you to use some of its features. They help to authenticate users and prevent fraudulent use of user accounts. Without these Cookies, the services that you have asked for cannot be provided, and we only use these Cookies to provide you with those services.
```

> The cookies we use are listed below with details about why we use them and how long they will last. These essential cookies are used for site navigation and session management, and this service may not function correctly without them.

Reviewer: ```suggestion
The Cookies we use are listed below with details about why we use them and how long they will last. These essential Cookies are used for site navigation and session management, and this service may not function correctly without them.
```

| Name                                          | Purpose                                                                                                                                                                                                                                                  | Expires                                         |
| --------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------------|

> | `_dfe_session`                                | temporary session data                                                                                                                                                                                                                                   | End of browser session                          |

Reviewer: ```suggestion
| `_dfe_session`                                | Temporary session data                                                                                                                                                                                                                                   | End of browser session                          |
```

> | `__zlcmid`                                    | Zendesk cookie that identifies your webchat session                                                                                                                                                                                                      | 1 year                                          |

Reviewer: ```suggestion
| `__zlcmid`                                    | Zendesk Cookie that identifies your webchat session                                                                                                                                                                                                      | 1 year                                          |
```

> | `__zlcprivacy`                                | Zendesk cookie that stores your webchat cookie preferences                                                                                                                                                                                               | 1 year                                          |

Reviewer: ```suggestion
| `__zlcprivacy`                                | Zendesk Cookie that stores your webchat cookie preferences                                                                                                                                                                                               | 1 year                                          |
```

> | `git-cookie-preferences-v1`                   | Website cookie that stores your cookie preferences                                                                                                                                                                                                       | 90 days                                         |

Reviewer: ```suggestion
| `git-cookie-preferences-v1`                   | Website Cookie that stores your Cookie preferences                                                                                                                                                                                                       | 90 days                                         |
```

> | `_vis_opt_exp_{CAMPAIGNID}_goal_{GOALID}`     | VWO Cookie created when a campaign goal is triggered. This cookie, when created, will always have a value of 1                                                                                                                                   | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_goal_{GOALID}`     | VWO Cookie created when a campaign goal is triggered. This Cookie, when created, will always have a value of 1                                                                                                                                   | 100 days                                        |
```

> | `_vis_opt_exp_{CAMPAIGNID}_goal_{GOALID}`     | This VWO cookie is created when a campaign goal is triggered. This cookie, when created, will always have a value of 1                                                                                                                                   | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_goal_{GOALID}`     | VWO Cookie created when a campaign goal is triggered. This cookie, when created, will always have a value of 1                                                                                                                                   | 100 days                                        |
```

> | `_vis_opt_test_cookie`                        | VWO Cookie that reads if cookies can be placed                                                                                                                                                                                                      | Browser closes                                  |

Reviewer: ```suggestion
| `_vis_opt_test_cookie`                        | VWO Cookie that reads if Cookies can be placed                                                                                                                                                                                                      | Browser closes                                  |
```

> | `_vis_opt_test_cookie`                        | This VWO cookie is to read if cookies can be placed                                                                                                                                                                                                      | Browser closes                                  |

Reviewer: ```suggestion
| `_vis_opt_test_cookie`                        | VWO Cookie that reads if cookies can be placed                                                                                                                                                                                                      | Browser closes                                  |
```

> | `_vis_opt_exp_{CAMPAIGNID}_combi`             | This VWO cookie is created when you are chosen for a particular variation for a test and ensures you see the same variation when you revisit a page in the future                                                                                        | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_combi`             | VWO Cookie created when you are chosen for a particular variation for a test and ensures you see the same variation when you revisit a page in the future                                                                                        | 100 days                                        |
```

> | `_vis_opt_exp_{CAMPAIGNID}_exclude`           | This VWO cookie is created when you are excluded from a test campaign due to the traffic percentage specifications                                                                                                                                       | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_exclude`           | VWO Cookie created when you are excluded from a test campaign due to the traffic percentage specifications                                                                                                                                       | 100 days                                        |
```

> | `_vis_opt_exp_{CAMPAIGNID}_split`             | This cookie is created when one of the variations is chosen for you, but you are not yet redirected to the variation page. It exists so that VWO can verify that the variation page corresponds to the chosen variation after landing on the page        | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_split`             | VWO Cookie created when one of the variations is chosen for you, but you are not yet redirected to the variation page. It exists so that VWO can verify that the variation page corresponds to the chosen variation after landing on the page        | 100 days                                        |
```

> | `_vis_opt_s`                                  | This VWO cookie tracks sessions                                                                                                                                                                                                                          | 100 days                                        |

Reviewer: ```suggestion
| `_vis_opt_s`                                  | VWO Cookie that tracks sessions                                                                                                                                                                                                                          | 100 days                                        |
```

> | `_vis_opt_out`                                | This VWO cookie indicates that you should not be made part of any campaign                                                                                                                                                                               | 10 years                                        |

Reviewer: ```suggestion
| `_vis_opt_out`                                | VWO Cookie that indicates that you should not be made part of any campaign                                                                                                                                                                               | 10 years                                        |
```

> | `_vwo_uuid`                                   | This VWO cookie generates a unique ID for every visitor                                                                                                                                                                                                  | 10 years                                        |

Reviewer: ```suggestion
| `_vwo_uuid`                                   | VWO Cookie that generates a unique ID for every visitor                                                                                                                                                                                                  | 10 years                                        |
```

> | `_vwo_ds`                                     | VWO cookie that stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |

Reviewer: ```suggestion
| `_vwo_ds`                                     | VWO Cookie that stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |
```

> | `_vwo_ds`                                     | This VWO cookie stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |

Reviewer: ```suggestion
| `_vwo_ds`                                     | VWO cookie that stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |
```

> | `_vwo_sn`                                     | VWO cookie that stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |

Reviewer: ```suggestion
| `_vwo_sn`                                     | VWO Cookie that stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |
```

> | `_vwo_sn`                                     | This VWO cookie stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |

Reviewer: ```suggestion
| `_vwo_sn`                                     | VWO cookie that stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |
```

> | `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | VWO Cookie that stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | VWO Cookie that stores the chosen combination. It's a temporary Cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |
```

> | `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | This VWO cookie stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |

Reviewer: ```suggestion
| `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | VWO Cookie that stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |
```

> | `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 secs                                         |

Reviewer 2: ```suggestion
| `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |
```
should we be using the full word rather than the abbreviation, like we do in the section below? same for 'mins'?

> | `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |

Reviewer: ```suggestion
| `_vwo_referrer`                               | VWO Cookie that stores referral information allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |
```

> | `_vvwo`                                       | This VWO cookie is a jar of all the VWO cookies. It is currently created only for the Safari browser                                                                                                                                                     | Expires in 2096 years                           |

Reviewer: ```suggestion
| `_vvwo`                                       | VWO Cookie that is a jar of all the VWO Cookies. It is currently created only for the Safari browser                                                                                                                                                     | Expires in 2096 years                           |
```

## Tracking and performance cookies

> **Type:** Persistent cookies

Reviewer: ```suggestion
**Type:** Persistent Cookies
```

**Administered by:** Third parties

> **Purpose:** These cookies are used to track information about traffic to the website and how users use the website.  We measure website usage so that we can gain a better understanding of how people use our site. This enables us to improve the information that we provide. It also ensures that the service is available when you want it, and that it delivers information quickly.

Reviewer: ```suggestion
**Purpose:** These Cookies are used to track information about traffic to the Website and how users use the Website.  We measure Website usage so that we can gain a better understanding of how people use our site. This enables us to improve the information that we provide. It also ensures that the service is available when you want it, and that it delivers information quickly.
```

> The information gathered via these cookies may directly or indirectly identify you as an individual visitor. This is because the information collected is typically linked to a pseudonymous identifier associated with the device you use to access the website. We may also use these cookies to test new pages, features or new functionality of the website to see how people react to them.

Reviewer: ```suggestion
The information gathered via these Cookies may directly or indirectly identify you as an individual visitor. This is because the information collected is typically linked to a pseudonymous identifier associated with the device you use to access the Website. We may also use these cookies to test new pages, features or new functionality of the Website to see how people react to them.
```

| Name                                                | Purpose                                                                                                                                                        | Expires after                                              |
| ------                                              | --------                                                                                                                                                       | -------                                                    |
| `bam-session`                                       | BAM use this cookie to attribute mailing list sign ups back to the agent at a teaching event that presented a QR code to the candidate                         | 30 days                                                    |

> | `_ga`                                               | Google Analytics cookie that helps us count how many people visit our website by tracking if you've visited before                                             | 2 years                                                    |

Reviewer: ```suggestion
| `_ga`                                               | Google Analytics Cookie that helps us count how many people visit our Website by tracking if you've visited before                                             | 2 years                                                    |
```

> | `_gid`                                              | Google Analytics cookie that helps us count how many people visit our website by tracking if you've visited before                                             | 24 hours                                                   |

Reviewer: ```suggestion
| `_gid`                                              | Google Analytics Cookie that helps us count how many people visit our Website by tracking if you've visited before                                             | 24 hours                                                   |
```

> | `_gat`                                              | Google Analytics cookie that helps us understand how people use our website                                                                                    | 1 minute                                                   |

Reviewer: ```suggestion
| `_gat`                                              | Google Analytics Cookie that helps us understand how people use our Website                                                                                    | 1 minute                                                   |
```

| `AMP_TOKEN`                                         | Uses tokens to help identify your client ID                                                                                                                    | 30 seconds to 1 year                                       |
| `_ga_76JLPNZBCP` `_ga_2BHBCSRT7L` `_ga_D3DXWHT5D3`  | Used by Google Analytics to find and track an individual session with your device                                                                              | 2 years                                                    |
| `_gat_UA-179982899-1`                               | Used by Google Analytics to find and track an individual session with your device                                                                              | 1 minute                                                   |

> | `_gaexp`                                            | Used by Google Optimize to identify if you've been part of an experiment or previous experiments which help us test parts of the website                       | Depends on the length of experiment, but typically 90 days |

Reviewer 2: Are we still using Google Optimize? is this needed?

Reviewer: We need to leave it in for now as it is still on the site - we have a separate ticket to get rid of it

> | `_gaexp`                                            | Used by Google Optimize to identify if you've been part of an experiment or previous experiments which help us test parts of the website                       | Depends on the length of experiment, but typically 90 days |

Reviewer: ```suggestion
| `_gaexp`                                            | Used by Google Optimize to identify if you've been part of an experiment or previous experiments which help us test parts of the Website                       | Depends on the length of experiment, but typically 90 days |
```

> | `_opt_utmc`                                         | Used by Google Optimize to store information about the source of website traffic                                                                               | 24 hours                                                   |

Reviewer: ```suggestion
| `_opt_utmc`                                         | Used by Google Optimize to store information about the source of Website traffic                                                                               | 24 hours                                                   |
```

| `_opt_expid`                                        | Used by Google Optimize to store information about experiments involving redirected pages                                                                      | 10 seconds                                                 |
| `_clck`                                             | Persists the Clarity user ID and preferences, unique to that site is attributed to the same user ID                                                            | 13 months                                                  |
| `_clsk`                                             | Connects multiple page views by a user into a single Clarity session recording                                                                                 | 13 months                                                  |
| `CLID`                                              | Identifies the first-time Clarity saw this user on any site using Clarity                                                                                      | 13 months                                                  |

> | `ANONCHK`                                           | Indicates whether MUID is transferred to ANID, a cookie used for advertising                                                                                   | 13 months                                                  |

Reviewer: ```suggestion
| `ANONCHK`                                           | Indicates whether MUID is transferred to ANID, a Cookie used for advertising                                                                                   | 13 months                                                  |
```

| `MR`                                                | Indicates whether to refresh MUID                                                                                                                              | 13 months                                                  |

> | `MUID`                                              | Identifies unique web browsers visiting Microsoft sites. These cookies are used for advertising, site analytics, and other operational purposes                | 13 months                                                  |

Reviewer: ```suggestion
| `MUID`                                              | Identifies unique web browsers visiting Microsoft sites. These Cookies are used for advertising, site analytics, and other operational purposes                | 13 months                                                  |
```

| `SM`                                                | Used in synchronising the MUID across Microsoft domains                                                                                                        | 13 months                                                  |

## Marketing cookies

> **Type:** Persistent cookies

Reviewer: ```suggestion
**Type:** Persistent Cookies
```

> **Administered by:** Third parties

Reviewer: ```suggestion
**Administered by:** Third Parties
```

> **Purpose:** These cookies track your browsing habits to enable us to show advertising which is more likely to be of interest to you. These cookies use information about your browsing history to group you with other users who have similar interests. Based on that information, and with our permission, third party advertisers can place cookies to enable them to show adverts which we think will be relevant to your interests while you are on third party websites.

Reviewer: ```suggestion
**Purpose:** These Cookies track your browsing habits to enable us to show advertising which is more likely to be of interest to you. These Cookies use information about your browsing history to group You with other users who have similar interests. Based on that information, and with our permission, third party advertisers can place Cookies to enable them to show adverts which we think will be relevant to your interests while You are on third party websites.
```

> We place marketing tags on our pages to track the success of our content on each channel. Some of our pages may contain content from other sites, like YouTube, which may set their own cookies. These sites are sometimes called ‘third party’ services. This tells us how many people are seeing the content and whether it’s useful. In addition, if you share a link to one of our pages, the service you share it on (for example, Facebook) may set a cookie. We have no control over cookies set on other websites - you can turn them off, but not through us.

Reviewer: ```suggestion
We place marketing tags on our pages to track the success of our content on each channel. Some of our pages may contain content from other sites, like YouTube, which may set their own Cookies. These sites are sometimes called ‘third party’ services. This tells us how many people are seeing the content and whether it’s useful. In addition, if you share a link to one of our pages, the service you share it on (for example, Facebook) may set a Cookie. We have no control over Cookies set on other websites - you can turn them off, but not through us.
```

| `_gac_UA-179982899-1`| Records how you’ve interacted with advertising campaigns                                                                                                              | 90 days                |
| `_gac_gb_76JLPNZBCP` | Used by Google Analytics and Google Ads to help us understand how you've interacted with advertising campaigns                                                        | 90 days                |
| `_gcl_au`            | Used by Google Analytics and Google Ads to help us understand how you've interacted with advertising campaigns                                                        | 90 days                |
| `_fbp`               | Used by Facebook to store and track your visits across websites                                                                                                       | 90 days                |
| `_fbc`               | Used by Facebook to record that you’ve accessed our website using a Facebook advert                                                                                   | 90 days                |

## Your choices regarding cookies

> If you prefer to avoid the use of cookies on the website, you must disable the use of cookies in your browser and then delete the cookies saved in your browser associated with this website. You may use this option for preventing the use of cookies at any time.

Reviewer: ```suggestion
If you prefer to avoid the use of Cookies on the Website, you must disable the use of Cookies in your browser and then delete the Cookies saved in your browser associated with this Website. You may use this option for preventing the use of Cookies at any time.
```

> If you do not accept our cookies, you may experience some inconvenience in your use of the website and some features may not function properly.

Reviewer: ```suggestion
If you do not accept our Cookies, you may experience some inconvenience in your use of the Website and some features may not function properly.
```

> If you'd like to delete cookies or instruct your web browser to delete or refuse cookies, please visit the help pages of your web browser: 

Reviewer: ```suggestion
If you'd like to delete Cookies or instruct your web browser to delete or refuse Cookies, please visit the help pages of your web browser: 
```
